### PR TITLE
WT-7072 Column store example

### PR DESCRIPTION
**Pull request by Sean04**
ex_col_store.c provides an example use case of the column-store API. A table is defined with columns to store weather data and column groups to allow multiple columns to be accessed through a single cursor. The data is randomly populated and example operations are performed on the table:
Op 1: Find average data given a location
Op 2: Find max/min temperature data given a time range
Op 3: Update temperature from Celsius to Fahrenheit
Op 4: Calculate chance of rain given humidity and pressure
Op 5: Remove the data for a given country

### CODE REVIEW

**Review by tammybailey — APPROVED** [review]
Added some suggestions for making the output more interesting but LGTM otherwise!

File: examples/c/ex_col_store.c [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+    /* If the min/max temperature is not found due to some error, there is no result to print. */
+    if (ret == 0) {
+        printf("The minimum temperature between %" PRIu16 " and %" PRIu16 " is %d.\n",
+          starting_time, ending_time, min_temp_result);
+        printf("The maximum temperature between %" PRIu16 " and %" PRIu16 " is %d.\n",
+          starting_time, ending_time, max_temp_result);
+    }
+
+    remove_country(session);
+    average_data(session);
```
[the remark this answers is not in the document]
Comment (Sean04) [reply examples/c/ex_col_store.c]: I updated it and added a loop in the main function to call average_data for each country.

File: examples/c/ex_col_store.c [hunk 2]
```diff
@@ -0,0 +1,484 @@
...
+     * Example uses 10am - 8pm but can change the values for desired start and end times.
+     */
+    starting_time = 1000;
+    ending_time = 2000;
+    min_temp_result = 0;
+    max_temp_result = 0;
+    ret = find_min_and_max_temp(
+      session, starting_time, ending_time, &min_temp_result, &max_temp_result);
+
+    /* If the min/max temperature is not found due to some error, there is no result to print. */
```
[the remark this answers is not in the document]
Comment (Sean04) [reply examples/c/ex_col_store.c:472]: Added.

File: examples/c/ex_col_store.c [hunk 3]
```diff
@@ -0,0 +1,484 @@
...
+    if (ret == 0) {
+        printf("The minimum temperature between %" PRIu16 " and %" PRIu16 " is %d.\n",
+          starting_time, ending_time, min_temp_result);
+        printf("The maximum temperature between %" PRIu16 " and %" PRIu16 " is %d.\n",
+          starting_time, ending_time, max_temp_result);
+    }
+
+    remove_country(session);
```
[the remark this answers is not in the document]
Comment (Sean04) [reply examples/c/ex_col_store.c:506]: I added the remove print and called average_data before and after to demonstrate that country was removed.

**Pull request merged**